.github/workflows/run-tests.yml

```diff
@@ -2,21 +2,20 @@ name: run-tests
 
 on:
   push:
-    branches: [main]
+    branches: [ main ]
   pull_request:
-    branches: [main]
-  workflow_dispatch:
+    branches: [ main ]
 
 jobs:
   test:
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: true
       matrix:
-        os: [ubuntu-latest]
-        php: [8.2]
-        laravel: [11.*]
-        stability: [prefer-lowest, prefer-stable]
+        os: [ ubuntu-latest ]
+        php: [ 8.2 ]
+        laravel: [ 10.*,11.* ]
+        stability: [ prefer-lowest, prefer-stable ]
```

Comment:
Might as well remove stability.
We're not using it so it's just doing extra test runs for no reason.